client/app/dedicatedCloud/dashboard/main/tiles/options/servicePack/service.js

```diff
@@ -0,0 +1,186 @@
+import _ from 'lodash';
+
+import { OPTION_TYPES } from './option/constants';
+import { PREFERENCE_NAME } from '../preference/constants';
+
+export default class DedicatedCloudServicePack {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    DedicatedCloud,
+    servicePackOptionService,
+    OvhHttp,
+    ovhUserPref,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.DedicatedCloud = DedicatedCloud;
+    this.servicePackOptionService = servicePackOptionService;
+    this.OvhHttp = OvhHttp;
+    this.ovhUserPref = ovhUserPref;
+  }
+
+  fetchNamesForService(serviceName) {
+    return this.OvhHttp.get(`/dedicatedCloud/${serviceName}/servicePacks`, {
```

Comment:
No `ovh-api-services` for this one ? Imo you should try and use only one library for API calls, it will be easier to migrate if we need to

Reply:
I'll do it !